test/fuzz/transformation_add_parameter_test.cpp

```diff
@@ -188,18 +204,19 @@ TEST(TransformationAddParameterTest, NonPointerBasicTest) {
          %41 = OpTypeStruct %11 %16
          %42 = OpConstantComposite %41 %8 %32
          %44 = OpTypeFunction %2 %41 %7
-          %6 = OpTypeFunction %7 %7 %11
+         %6 = OpTypeFunction %7 %7 %11 %52
```

Comment:
Nit: adjust whitespace so that the `=` symbols line up (that's the way the test was formatted before).

Reply:
Done